src/utils/parsers/currency.js

```diff
@@ -0,0 +1,27 @@
+export const centsToReal = (cents) => {
+  if (typeof cents !== 'number' || !Number.isInteger(cents)) {
+    throw new TypeError('cents must be an Integer number');
+  }
+
+  return (cents / 100).toLocaleString('pt-br', {
```

Comment:
I feel like this `100` here is a Magic Number. It has the same meaning here and in line 24, but is not explicit what they mean. 

If for some reason we have to change this value in the future, it might happen that someone forgets to change in both places.

So my suggestion is to create a const with a name that describes the meaning of this value, and we can use it in this line and in line 24, does it make sense?

An article about Magic Numbers: https://codetuts.dev/magic-numbers-in-programming/